### PR TITLE
Practice shape

### PR DESCRIPTION
**Pull request by lourdesVillca**

### CODE REVIEW

**Comment by lourdesVillca** [comment]
@carledriss ready for code review

File: Shapes/src/test/java/org/fundacionjala/org/SquareTest.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package org.fundacionjala.org;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
```
Comment (carledriss) [inline Shapes/src/test/java/org/fundacionjala/org/SquareTest.java]: Don't use the *

**Comment by carledriss** [comment]
It is not necessary to commit Shapes/ folder
directly commit src folder and pom.xml

**Comment by carledriss** [comment]
@lourdesVillca 
Please update the packages

**Comment by lourdesVillca** [comment]
@carledriss ready to code review, I delete the shape folder and now there is only the src folder

**Comment by carledriss** [comment]
Looks good to me!
:)

**Pull request merged**